### PR TITLE
add introduction notebook for jax

### PR DESCRIPTION
**Pull request by metaboulie**
## 📋 Checklist

- [ ] I have read the [contributor guidelines](../CONTRIBUTING.md).
- [ ] For large changes, or changes that affect the public API: this change was discussed or approved through an issue, on [Discord](https://discord.gg/JE7nhX6mD8), or the community [discussions](https://github.com/marimo-team/marimo/discussions) (Please provide a link if applicable).
- [ ] I have added tests for the changes made.
- [ ] I have run the code and verified that it works as expected.

## 📜 Reviewers

@Haleshot OR @metaboulie

### CODE REVIEW

**Comment by metaboulie** [comment]
@Haleshot  have a look when you are free :)

**Comment by Haleshot** [comment]
> @Haleshot have a look when you are free :)

Will do!

File: marimo-tutorials/Data-Science/Jax/introduction.py [hunk 1]
```diff
@@ -0,0 +1,782 @@
...
+
+
+        def update(params, x, y, epoch_number):
+          grads = grad(loss)(params, x, y)
+          lr = INIT_LR * DECAY_RATE ** (epoch_number / DECAY_STEPS)
+          return [(w - lr * dw, b - lr * db)
+                  for (w, b), (dw, db) in zip(params, grads)]
+        ```
+        """
+    ).callout(kind="info")
```
Comment (Haleshot) [inline marimo-tutorials/Data-Science/Jax/introduction.py:519]: Really like this callout!

**Review by Haleshot — CHANGES_REQUESTED** [review]
The notebook gives great insights on Jax (atleast for a beginner like me!). Do let me know if you need in working on the suggestions offered in the review (following the [`init.py`](https://github.com/Haleshot/marimo-tutorials/blob/main/init.py)).

**Review by Haleshot — COMMENTED** [review]
Also, can we change the name of the notebook | location of the notebook to somewhere better? Since JAX is along the lines of PyTorch, Tensorflow, it can belong somewhere in [`marimo-tutorials\artificial-intelligence\deep-learning`](https://github.com/Haleshot/marimo-tutorials/tree/main/marimo-tutorials/artificial-intelligence/deep-learning)? What do you think?

File: marimo-tutorials/Data-Science/Jax/introduction.py [hunk 2]
```diff
@@ -0,0 +1,782 @@
...
+
+import marimo
+
+__generated_with = "0.9.10"
+app = marimo.App()
+
+
+@app.cell(hide_code=True)
+def __(mo):
+    mo.md(r"""# An Introduction to Jax""")
```
[the remark this answers is not in the document]
Comment (metaboulie) [reply marimo-tutorials/Data-Science/Jax/introduction.py:21]: yeah, I realized that I forgot to add the header

**Comment by metaboulie** [comment]
Thanks for the review
To sum up, I will add more hyperlinks, the header, and move it to `marimo-tutorials\attificial-intelligence\deep-learning\jax`

**Pull request merged**